<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>597</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1001</width>
    <height>597</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1001</width>
    <height>597</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tower of Hanoi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/images/icon.png</normaloff>:/new/images/icon.png</iconset>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>597</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="front">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>597</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/new/images/front.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>520</y>
       <width>195</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 30px;
	background:  rgb(116,93,58);
	font-weight: bold;
	border: 2px solid;
	border-radius: 10px;
}

*:hover{
	font-size: 33px;
	background: black;
	color:  rgb(116,93,58);
}</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LeaveButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>520</y>
       <width>195</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 30px;
	background:  rgb(116,93,58);
	font-weight: bold;
	border: 2px solid;
	border-radius: 10px;
}

*:hover{
	font-size: 33px;
	background: black;
	color:  rgb(116,93,58);
}</string>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1001</width>
       <height>600</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/images/background.png);</string>
     </property>
    </widget>
    <widget class="QLabel" name="movementCounterLabel">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>16</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 20px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="nextLevelButton">
     <property name="geometry">
      <rect>
       <x>792</x>
       <y>400</y>
       <width>193</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 30px;
	background:  rgb(116,93,58);
	font-weight: bold;
	border: 2px solid;
	border-radius: 10px;
}

*:hover{
	font-size: 33px;
	background: black;
	color:  rgb(116,93,58);
}</string>
     </property>
     <property name="text">
      <string>Next Level</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="resetLevelButton">
     <property name="geometry">
      <rect>
       <x>792</x>
       <y>310</y>
       <width>193</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 30px;
	background:  rgb(116,93,58);
	font-weight: bold;
	border: 2px solid;
	border-radius: 10px;
}

*:hover{
	font-size: 32px;
	background: black;
	color:  rgb(116,93,58);
}</string>
     </property>
     <property name="text">
      <string>Reset Level</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>17</y>
       <width>130</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 20px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Movements:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>323</x>
       <y>11</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	border: 2px solid;
	border-radius: 15px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>792</x>
       <y>220</y>
       <width>193</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 23px;
	background:  rgb(116,93,58);
	font-weight: bold;
	border: 2px solid;
	border-radius: 10px;
}

*:hover{
	font-size: 24px;
	background: black;
	color:  rgb(116,93,58);
}</string>
     </property>
     <property name="text">
      <string>Return to menu</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>120</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 35px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="levelCounterLabel">
     <property name="geometry">
      <rect>
       <x>917</x>
       <y>117</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 35px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="wellDoneLabel">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>270</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 60px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Well Done!!</string>
     </property>
    </widget>
    <widget class="QLabel" name="endGameLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>340</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	font-size: 60px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QLabel" name="invalidMovementLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	font-family: &quot;MathJax_Fraktur&quot;;
	color:red;
	font-size: 40px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Invalid Movement</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
